### ISSUE TITLE
how to install and update packages using admin for normal scoop

### ISSUE THREAD

**Issue by candrapersada** [ScoopInstaller/Nonportable#13]
how to install and update packages admin for normal scoop like notepadplusplus-np?

**Comment by issaclin32** [comment]
Run your command-line / Powershell under admin.

Or, you can install sudo (`scoop install sudo`), and run `sudo scoop install notepadplusplus-np`.

**Comment by candrapersada** [comment]
why command-line under admin get error message `ERROR Exit code was 1620!`

**Comment by issaclin32** [comment]
> is it possible to update the package without admin `scoop update *`

It depends on whether the **installer exe** requires admin rights or not. For most non-portable packages, the answer is **no**.

**Comment by issaclin32** [comment]
> is it possible to update the package without admin `scoop update *`

Yes. If you run your command line **under admin**, there will be no problem updating these packages.
You can also run `sudo scoop update *`

By the way, most non-portable packages requires admin rights to run because either
(1) the **installer exe** requires admin rights to run,
 or (2) The installer will only work properly with admin rights on.

**Comment by candrapersada** [comment]
is it safe to update non-admin packages using admin?

**Comment by issaclin32** [comment]
> is it safe to update non-admin packages using admin like portable packages?

non-admin packages will work fine. It doesn't have any difference whether you install it **under admin rights** or not.

**Comment by candrapersada** [comment]
why is there no package list information that uses admin
![image](https://user-images.githubusercontent.com/36833664/175836035-b5f4bcb7-5c0e-475a-ae73-51bbb15fa706.png)

**Comment by issaclin32** [comment]
> why is there no package list information that uses admin

Will add some instructions later.
